util.js

```diff
@@ -640,4 +640,20 @@
         return url && url.replace(/^https/, "http");
     };
 
+    /**
+     * Unescapes HTML entities so the text in which they appear can be displayed as text of DOM elements
+     * 
+     * This implementation is as suggested by the MDN:
+     * https://developer.mozilla.org/en-US/Add-ons/Code_snippets/HTML_to_DOM
+     *
+     * @param {String} html foto escape entities
+     * 
+     * @return {String}
+     */
+    DDG.unescape = function(text) {
+        var doc = document.implementation.createHTMLDocument("example");
```

Comment:
@andrey-p @bsstoner any thoughts on this approach?

@jagtalon found it IIRC and showed it to me:  https://developer.mozilla.org/en-US/Add-ons/Code_snippets/HTML_to_DOM


We're currently using it for the Couprex IA. The alternative is a more verbose approach like the one used by Underscore.js

http://underscorejs.org/docs/underscore.html#section-160